google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/apis/AddCloudLibrariesAction.java

```diff
@@ -35,6 +39,23 @@ public AddCloudLibrariesAction() {
         GoogleCloudToolsIcons.CLOUD);
   }
 
+  @Override
+  public void update(AnActionEvent e) {
+    boolean enableAddLibraries = false;
+    Project project = e.getProject();
+    if (project != null) {
+      // check if this project has any 'maven-style' modules.
+      for (Module module: ModuleManager.getInstance(project).getModules()) {
```

Comment:
`DefaultAppEngineProjectService` has an isMavenModule method. My point isn't that we should use that instead, but I see that it also does a check for `MavenProjectsManager#findProject` not being null. I'm wondering if that is redundant.

Reply:
Hard to say from the source code, but let's do that in one place, always a good thing.

Reply:
sure. I don't like that that is an "app engine" utility. But we can make note to refactor that.